test/e2e/usage_test.go

```diff
@@ -127,7 +127,7 @@ func TestUsageComposition(t *testing.T) {
 	}))
 
 	environment.Test(t,
-		features.New(t.Name()).
+		features.NewWithDescription(t.Name(), "Tests scenarios for Crossplane's `Usage` resource as part of a composition.").
```

Comment:
We mostly had a pattern in this PR of removing the function level comment because it was duplicated by this `features.NewWithDescription()` call, but it's not 100% consistent. Here's an example where the function level comment still remains.

Did you want to make it 100% consistent throughout this PR?

Reply:
This was an oversight, thanks for picking up on it.